### PR TITLE
scaffolder(next): implement the review step

### PR DESCRIPTION
**Pull request by benjdlambert**
- Implementing something like the current review screen
- Create button still left

ezgif-2-232fc6959a

### CODE REVIEW

File: .changeset/two-planets-provide.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder': major
```
Comment (jhaals) [inline .changeset/two-planets-provide.md:2]: major? 😮

Comment (benjdlambert) [reply .changeset/two-planets-provide.md]: oops, just wanted to create the changeset and write in the editor! :joy:

**Review by jhaals — APPROVED** [review]
Nice! 👍

**Comment by regicsolutions** [comment]
@benjdlambert is there any documentation around how to switch to using this new wizard view?

**Comment by benjdlambert** [comment]
Not as yet, as it's not working end to end yet. Hoping to have some in the 1.7 release.

**Pull request merged**